contracts/Requests.sol

```diff
@@ -36,3 +36,27 @@ struct PoR {
   bytes publicKey; // public key
   bytes name; // random name
 }
+
+library Requests {
```

Comment:
Great suggestion, thanks! I tried creating the Conversions library, but it ended up holding only `toRequestIds()` and `toSlotIds()`; the other conversions were no longer needed. In the end I decided to keep everything in the `Requests` library, because I figured that things related to RequestId and SlotId are strongly related to requests. I moved the `slotId()` function there as well, as you suggested.